stock_move_description/stock_move.py

```diff
@@ -19,44 +19,42 @@
 #
 ##############################################################################
 
-from openerp.osv import orm, fields
+from odoo import fields, models
 
 
-class StockMove(orm.Model):
+class StockMove(models.Model):
     _inherit = "stock.move"
+    name = fields.Text(string="Description", required=False, )
 
-    _columns = {
-        'name': fields.text('Description'),
-    }
 
     def onchange_product_id(
```

Comment:
@AbdulrhmanGad If it is an onchange function, it must be decorated with the good api.onchange().